event/floating_continent.py

```diff
@@ -35,6 +37,8 @@ def mod(self):
             self.ground_character_mod(self.reward1.id)
         elif self.reward1.type == RewardType.ESPER:
             self.ground_esper_mod(self.reward1.id)
+        elif self.reward1.type == RewardType.ITEM:
+            self.ground_item_mod(self.reward1.id)
```

Comment:
Floating continent 1/3 can now yield items (not by default, but they can be overwritten using the new flags)